guidelines/terms/20/image-of-text.html

```diff
@@ -1,11 +1,11 @@
 <dt><dfn id="dfn-images-of-text" data-lt="images of text">image of text</dfn></dt>
 <dd>
    
-   <p>text that has been rendered in a non-text form (e.g., an image) in order to achieve
+  <p>text that has been rendered in a non-text form (e.g., an image) in order to achieve
```

Comment:
It is not clear why this paragraph is marked as a change.  I do not even find white space differences.

Reply:
There is a leading whitespace change before the `<p>`.

Reply:
Now I see the white space difference!